src/envo/envo-edit.owl

```diff
@@ -31615,76 +31633,133 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001406> ObjectSomeValuesFrom(<
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01001412> "stratocumulus cloud"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001412> <http://purl.obolibrary.org/obo/ENVO_01001428>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001412> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/ENVO_03520000>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001412> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002473> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/BFO_0000027> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002351> ObjectUnionOf(<http://purl.obolibrary.org/obo/ENVO_01000832> <http://purl.obolibrary.org/obo/ENVO_01000833> <http://purl.obolibrary.org/obo/ENVO_01001876> <http://purl.obolibrary.org/obo/ENVO_03520007>)))))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001412> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002473> ObjectUnionOf(<http://purl.obolibrary.org/obo/ENVO_00002006> <http://purl.obolibrary.org/obo/ENVO_01000277>)))
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_01001413> (stratus cloud)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/clouds-genera-stratus.html") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/clouds-genera.html") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/docs/Excerpt%20from%20wmo_49-1_final_2017.pdf") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Cloud") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01001413> "A cloud which is 1) generally grey in colour, 2) structurally organised as a layer  with a fairly uniform base or, less frequently, ragged patches, 3) disposed to produce drizzle, snow or snow grains as preciptation, 4) only produces halo phenomena at very low temperatures, and 5) allows a clearly visible outline of the local star to be seen through its mass.")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/clouds-genera-stratus.html") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/clouds-genera.html") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://cloudatlas.wmo.int/docs/Excerpt%20from%20wmo_49-1_final_2017.pdf") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Cloud") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01001413> "A cloud which 1) constitutes a layer with a fairly uniform base or, less frequently, ragged patches, 2) is composed of small water droplets, 3) is located in the low level of the atmosphere (low étage)."@en)
```

Comment:
"constitutes" see above. Note this elsewhere too, as I won't be commenting on it further.